website/src/word-panel.css.tsx

```diff
@@ -81,3 +81,11 @@ export const wordPanelHeader = style({
 })
 
 export const audioContainer = style({ paddingLeft: "40%" })
+
+export const table = style({
```

Comment:
I feel like the table itself and the table cells probably need different style classes.